qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -287,7 +287,7 @@ public void endPatientVisit() {
 
 	@Then("the system ends the patient visit")
 	public void systemEndsPatientVisit() {
-		assertNull(visitsDashboardPage.getActiveVisit());
+		assertTrue(textExists("No active visit"));
```

Comment:
thought we had changed to use something different? :smiley:

Reply:
> thought we had changed to use something different? smiley

@sherrif10  from this [conversation](https://github.com/openmrs/openmrs-contrib-qaframework/pull/256#issuecomment-988907267) and what followed after, seemed like it was agreed to go with texts as opposed to pages.